src/main/java/com/chargerrobotics/subsystems/ShooterHoodSubsystem.java

```diff
@@ -57,6 +57,43 @@ public boolean isLimitSwitchTriggered() {
     public double getHoodPosition() {
         return (double)shooterHood.getSensorCollection().getQuadraturePosition();
     }
+    
+    public double getHoodAngle()  {
```

Comment:
Could you rename this to `getDesiredHoodAngle`? Otherwise, this suggests that this method returns the current hood angle.